src/replica/replica_stub.cpp

```diff
@@ -425,6 +431,7 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
     _mem_release_max_reserved_mem_percentage = _options.mem_release_max_reserved_mem_percentage;
     _max_concurrent_bulk_load_downloading_count =
         _options.max_concurrent_bulk_load_downloading_count;
+    _abnormal_write_trace_latency_threshold = FLAGS_abnormal_write_trace_latency_threshold;
```

Comment:
There is no need to add a member `_abnormal_write_trace_latency_threshold` for `replica_stub`. use `FLAGS_abnormal_write_trace_latency_threshold` directly

Reply:
`abnormal_write_trace_latency_threshold` need used by other cpp file. if del _abnormal_write_trace_latency_threshold, we need extern

Reply:
so you can use extern